### DISCUSSION TITLE
EPS Meteograms with Herbie

### DISCUSSION THREAD

**Discussion by csteele2** [blaylockbk/Herbie#131] [Show and tell]
Just sharing one of the things I have done with Herbie, and that is ensemble meteograms with EPS precipitation and temperatures. My messy python code is [here](https://github.com/csteele2/Wx4Colab/blob/master/EPS_Meteograms_with_Herbie.ipynb) (one does need a synoptic token, but only to look up site metadata; a dictionary can be easily coded in its place).
![EPS20221228-00_pcp24_KBLU](https://user-images.githubusercontent.com/110204437/210023933-c94650d9-0b91-42bf-ba0f-211907f6e480.png)

**Comment by blaylockbk** [comment]
Nice! That is a beautiful looking plot. I like how you show the individual members alongside the box plots.

**Reply by csteele2** [reply]
It is a nice visualization to concisely see ensemble agreement on timing and magnitude. And add that context to the extreme forecast index/shift-of-tails. I guess I need to read what assumption seaborn violin plots are making, because they do not look as I expect in place of the box-and-whiskers (but I would rather have something like that since precipitation is almost never gaussian).
